src/core/public/rendering/app_containers.test.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { BehaviorSubject } from 'rxjs';
+import { act } from 'react-dom/test-utils';
+import { mount } from 'enzyme';
+import React from 'react';
+
+import { AppWrapper, AppContainer } from './app_containers';
+
+describe('AppWrapper', () => {
+  it('toggles the `hidden-chrome` class depending on the chrome visibility state', () => {
+    const chromeVisible$ = new BehaviorSubject<boolean>(true);
+
+    const component = mount(<AppWrapper chromeVisible$={chromeVisible$}>app-content</AppWrapper>);
+    expect(component.html()).toMatchInlineSnapshot(
+      `"<div class=\\"app-wrapper\\">app-content</div>"`
+    );
```

Comment:
snapshot of the component is unnecessarily long as it display the whole `chromeVisible$` subject property tree, so to keep inline snapshots for readability, I'm snapshoting the html instead.